RUFAS/output_manager.py

```diff
@@ -761,59 +766,88 @@ def save_variables(
             f"exclude_info_maps flag set to {exclude_info_maps}",
             info_map,
         )
-        graph_generator = GraphGenerator()
-        list_of_filter_files = self._list_txt_and_json_files_in_dir(dir_path)
+        list_of_filter_files = self._list_txt_and_json_files_in_dir(filters_dir_path)
         for filter_file in list_of_filter_files:
-            input_path = os.path.join(dir_path, filter_file)
-            filter_content = self._load_filter_file_content(input_path)
-            if "filters" not in filter_content.keys():
-                self.add_error(
-                    "Missing filters entry",
-                    f"'filters' does not exist in {filter_file}",
+            for _, supported_prefix in self.__supported_filter_types_prefixes.items():
+                if filter_file.startswith(supported_prefix):
+                    break
+            else:
+                self.add_warning(
+                    "invalid filter file prefix",
+                    f"{filter_file} prefix is not in {list(self.__supported_filter_types_prefixes.values())}",
                     info_map,
                 )
                 continue
```

Comment:
```suggestion
                if filter_file.startswith(supported_prefix):
                    break
                else:
                    self.add_warning(
                        "invalid filter file prefix",
                        f"{filter_file} prefix is not in {list(self.__supported_filter_types_prefixes.values())}",
                        info_map,
                    )
                    continue
```
Should if and else statements be on the same level of indentation?

Reply:
the else belongs to the for loop not the if statement.

Reply:
Saw reply to Joe, disregard this. Neat pythonic trick to know about though.